kanidmd/score/src/https/middleware.rs

```diff
@@ -39,16 +189,12 @@ impl<State: Clone + Send + Sync + 'static> tide::Middleware<State>
                 we should be loading, and should be really secure about that!
 
             */
-            // TODO: consider scraping the other js files that wasm-pack builds and including them too
             "content-security-policy",
             vec![
                 "default-src 'self'",
+                // TODO: have a dev/test mode where we can rebuild the hashes on page load, so when doing constant JS changes/rebuilds we don't have to restart the server every time. It'd be *terrible* to run in prod because of the constant disk thrashing, but nicer for devs.
```

Comment:
Or in dev, just don't send hashes at all .....

Reply:
yeah, I was pondering which way to go on it, I'll raise it as an issue as it's a side quest at this point

Reply:
Raised as #912